### DISCUSSION TITLE
Help with setting SFTPGo with Podman (permission problems)

### DISCUSSION THREAD

**Discussion by Froggy232** [drakkan/sftpgo#1613] [Q&A]
Hi,
I try to deploy a sftpgo container on a homeserv, but I have permission problem as I use rootless podman.
I have tried, so far, to : 

- create a new user (UID=1036)
- create a `sftpgo.container` file in `.config/containers/systemd`, containing the following : https://privatebin.alrhome.net/?6c0232bd081783cb#4s1f9vM9yjoCeHfDtNX6mwuEY9orokWeo4u3rHL9hp2C
- then, I have created two directory, `mkdir data` & `mkdir home`

When I try to launch the container, it fail with this error message : https://privatebin.alrhome.net/?64bfbf1718fd55b1#5AFpKxsGyuHw2QEhh1ExJpncrQ98qB1fg4DnGRouZ2uU
If I try to `chmod -R 777` the home folder, it work normally and I can create an admin user, but I of course don't want this solution.

Does someone sucessfully achieve to run SFTPGo with Podman rootless?
I can pay to thanks the one who gave the solution.

Thanks you!

**Comment by drakkan** [comment]
Never used podman myself, try setting [userns](https://docs.podman.io/en/v4.6.0/markdown/options/userns.container.html) or something similar. SFTPGo runs using uid and gid 1000

**Comment by Froggy232** [comment]
Thanks for your input, I will try to play with that.
If someone have another idea, I'm open.
Thanks you!
